eng/config/PublishData.json

```
@@ -213,6 +213,14 @@
       "vsBranch": "rel/d16.9",
       "vsMajorVersion": 16
     },
+    "release/dev16.10-vs-deps": {
+      "nugetKind": [ "Shipping", "NonShipping" ],
+      "version": "3.10.*",
+      "packageFeeds": "default",
+      "channels": [ "dev16.10p1" ],
```

should this also have "dev16.10" channel?
```suggestion
      "channels": [ "dev16.10", "dev16.10p1" ],
```